source/patterns/@aws-solutions-constructs/core/lib/input-validation.ts

```diff
@@ -97,6 +97,11 @@ export function CheckProps(propsObject: VerifiedProps | any) {
     errorFound = true;
   }
 
+  if (propsObject.existingTableObj && propsObject.existingTableInterface) {
```

Comment:
This isn't really suitable for here because there is no construct that accepts both these values. I think we want it in the buildDynamoTable function because we've seen the use of that function can be confusing and some developer may call it inappropriately in the future.

If a construct does accept both of these in the future, we have broken our design guidelines upstream of this point and have a more existential problem that input validation :-)